src/schema/nmdc.yaml

```
@@ -2532,6 +2542,17 @@ slots:
     exact_mappings:
       - OBI:0002475
   
+  is_stranded:
+    description: Is the (RNA) library stranded or non-stranded (unstranded). 
+    range: boolean
+    comments: 
+      - A value of true means the library is stranded, flase means non-stranded. 
+  
+  stranded_orientation:
+    description: Lists the strand orientiation for a stranded RNA library preparation.
+    range: StrandedOrentationEnum
```

I think there is a typo in the name of the enum:

```suggestion
    range: StrandedOrientationEnum
```